### DISCUSSION TITLE
Creating mathematical channels from topic data

### DISCUSSION THREAD

**Discussion by AndreasZetterlund** [foxglove/community#70] [Q&A]
Hi! I'm relative new to foxglove and I am trying to plot different data vs eachother. For example, plotting the difference between two variables would be an amazing feature for our project. When I saw the variable tab I assumed I could create my own variables from topics data but unfortunately it does not seem like it can handle topics data.

I might have missed something and it's already doable. Thanks and love the work you are doing

**Comment by jtbandes** [comment]
Hi, thanks for trying out Foxglove Studio!
For custom calculations, I would recommend you look at the Node Playground panel: https://foxglove.dev/docs/studio/panels/node-playground
